hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3031,7 +3031,7 @@ public String getRatisRoles() {
     List<ServiceInfo> serviceList;
     int port = omNodeDetails.getRatisPort();
     RaftPeer leaderId;
-    if (isRatisEnabled) {
+    if (isRatisEnabled && null != omRatisServer) {
```

Comment:
Else is redundant at the end, also change the string to `Ratis disable or server shutting down"?